stdlib/2/os/__init__.pyi

```diff
@@ -130,7 +133,14 @@ def setuid(uid: int) -> None: ...  # Unix only
 def strerror(code: int) -> str: ...
 def umask(mask: int) -> int: ...
 def uname() -> Tuple[str, str, str, str, str]: ...  # Unix only
-def unsetenv(varname: str) -> None: ...
+
+@overload
+def getenv(key: Text) -> Optional[str]: ...
```

Comment:
Yes.  In python 2 you can use `str` or `unicode`, but in Python 3 it can only be `str` (i.e not `bytes`)  so I was using  the fact that `str` gets promoted to unicode in Python 2 in order to get that effect.  Another option is to use `Union[str, Text]` here.  Does it make a big difference?